msteams-ui-styles-core/src/colors.ts

```diff
@@ -16,6 +16,10 @@ export const Colors = {
     gray12: '#f4f4f4',
     gray13: '#F8F8F9', // don't know
     gray14: '#F9F8F7',
+    gray15: '#EDEBE9',
```

Comment:
I took a look at the color ramp here:
![image](https://user-images.githubusercontent.com/29067893/57955396-b0f30200-78aa-11e9-8146-62e532cf24cd.png)

And I don't see any reference to these new gray colors you are adding. I think you might have misunderstood the numbering sequence here. The number for gray is how transparent it is; but we boil it down to a hex value.

If you open that Color.PNG file, you should see the color ramp:
![image](https://user-images.githubusercontent.com/29067893/57955452-e0097380-78aa-11e9-8945-b68eed6291ef.png)

You should run through that and update the correct colors here.

Reply:
@Srivma All the colors in the specs should resolve to one of the colors in the color ramp above. Don't add any new values that are not defined in the color set.

Reply:
Sure, Adrian, But green1 colour is not mentioned in colour ramp.

just now Resolved the colour issue by using the colour ramp colours and pushed code changes

Reply:
![exisiting-colors-updates](https://user-images.githubusercontent.com/50403816/57960457-67e09380-7926-11e9-8c66-0cefca1ca127.png)

as per discussed with Matt, changed existing code colours by using the given colour ramp